docs/INSTALL.md

```diff
@@ -81,10 +81,34 @@ Install Cmake (**3.18.1** minimum), Boost, Qt 5 and other dependencies.
 
 Installing *ccache* can speed up active development.
 
-_Ubuntu 20.04, 22.04:_
+_Ubuntu 20.04:_
 
 ```bash
-sudo apt-get update && sudo apt-get install -y \
+sudo apt update && sudo apt install -y \
```

Comment:
We avoid copy-paste in the documentation and in the code. It will be harder to support it later. If you didn't like my previously proposed approach, then maybe this one will be better:
- Share common instructions for both 20 and 22.
- Add a separate substep "install cmake" with two points: Ubuntu 20 and Ubuntu 22.